t/tasks/inventory/generic/batteries/acpiconf.t

```diff
@@ -32,18 +33,20 @@ my %testAcpiconfMerged = (
                 MANUFACTURER    => 'Lg',
                 NAME            => 'DELL 242WD6B',
                 SERIAL          => 58167,
-                VOLTAGE         => '7600'
+                VOLTAGE         => '7600',
+                REAL_CAPACITY   => '52288',
```

Comment:
Here REAL_CAPACITY is not extracted from dmidecode output, so the test fails here. I don't think dmidecode will report such info anyway, so just remove this line.